bundles/org.openhab.binding.senseenergy/src/main/java/org/openhab/binding/senseenergy/internal/handler/SenseEnergyBridgeHandler.java

```diff
@@ -92,7 +90,7 @@ public void initialize() {
     public void goOnline() {
         try {
             this.monitorIDs = api.initialize(config.email, config.password);
-        } catch (InterruptedException | TimeoutException | ExecutionException | SenseEnergyApiException e) {
+        } catch (SenseEnergyApiException e) {
```

Comment:
Why changed this, we prefer the catch of explicit exceptions over the generic `Exception`